src/main/java/org/sagebionetworks/bridge/sdk/utils/Utilities.java

```diff
@@ -81,4 +81,8 @@ public static boolean isValidUrl(String url) {
         }
         return urlValidator.isValid(url);
     }
+    
+    public static String unescapeJson(String json) {
```

Comment:
Since this is used in a test, I'd feel better if it was only in the test code and not the SDK. To me unescapeJson implies functionality beyond what this does